src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.m

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
In XCode, this file and the above file show up in the correct directory. Looks like when I duplicated and modified the CBXMLHandler for the `PreviousLookBrick`, I forgot to rename it even though I updated it's contents, and the XCode for some reason physically moved it to another directory while keeping it logically in the same directory as intended. I'm fixing this right now.